public/puzzle.php

```diff
@@ -1,66 +1,19 @@
 <?php
 
-require_once 'env.php';
-require_once 'polite.class.php';
+use FriendlyCaptcha\Captcha;
+use FriendlyCaptcha\Polite;
+
+require_once 'vendor/autoload.php';
```

Comment:
would just put that in the same folder - otherwise it might conflict with composer?

Reply:
The "conflict" is intended. If you use composer, you just replace the autoload.php by composer's version and do not need to adopt the require statements.